app/src/main/java/com/example/criminalintent/presentation/crimeDetail/CrimeDetailViewModel.kt

```diff
@@ -69,7 +69,7 @@ class CrimeDetailViewModel : ViewModel() {
         crimeRepository.updateCrime(crime)
     }
 
-    fun addOrUpdate(crime: Crime) {
+    fun addOrUpdate() {
```

Comment:
With the removal of the param, it's not clear what this method is adding or updating. So I would rename to `addOrUpdateCrime` to make things clear, but it's just a suggestion